[2025-09-05T16:29:44.808+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-05T16:29:44.835+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_finanzas_dag.extract manual__2025-09-05T16:29:38.072190+00:00 [queued]>
[2025-09-05T16:29:44.842+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_finanzas_dag.extract manual__2025-09-05T16:29:38.072190+00:00 [queued]>
[2025-09-05T16:29:44.843+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-05T16:29:44.859+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract> on 2025-09-05 16:29:38.072190+00:00
[2025-09-05T16:29:44.917+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=136) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-05T16:29:44.906+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_finanzas_dag', 'extract', 'manual__2025-09-05T16:29:38.072190+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/etl_finanzas_dag.py', '--cfg-path', '/tmp/tmp9gtmh1u3']
[2025-09-05T16:29:44.919+0000] {standard_task_runner.py:63} INFO - Started process 143 to run task
[2025-09-05T16:29:44.919+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask extract
[2025-09-05T16:29:45.101+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_finanzas_dag.extract manual__2025-09-05T16:29:38.072190+00:00 [running]> on host 2915bafccca2
[2025-09-05T16:29:45.273+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Freddy' AIRFLOW_CTX_DAG_ID='etl_finanzas_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-09-05T16:29:38.072190+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-05T16:29:38.072190+00:00'
[2025-09-05T16:29:45.278+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-05T16:29:45.388+0000] {logging_mixin.py:188} INFO - Datos extraidos correctamente:300 filas
[2025-09-05T16:29:45.415+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-05T16:29:45.416+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-05T16:29:45.424+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_finanzas_dag, task_id=extract, run_id=manual__2025-09-05T16:29:38.072190+00:00, execution_date=20250905T162938, start_date=20250905T162944, end_date=20250905T162945
[2025-09-05T16:29:45.461+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-05T16:29:45.481+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-05T16:29:45.487+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
